tests/python/requirements/PySafetyBearTest.py

```diff
@@ -60,3 +60,17 @@ def test_with_no_pinned_requirements(self):
         ) as check:
             self.check_validity(self.uut, ['foo', 'bar>2'])
             assert not check.called
+
+    def test_with_no_description(self):
```

Comment:
As this test module heavily uses mock, it has not seen the error which occurs when using the real `safety` code.

This is almost good enough, but it does not check that the `advisory` message was included in the exception, which is the only way to test your code changes with mock.